std/algorithm/comparison.d

```diff
@@ -1382,6 +1385,8 @@ Iterates the passed arguments and returns the minimum value.
 Params: args = The values to select the minimum from. At least two arguments
     must be passed, and they must be comparable with `<`.
 Returns: The minimum of the passed-in values.
+See_Also:
+    $(XREF algorithm,searching,minElement)
```

Comment:
Ah I already did this for `maxElement`, but just forgot it. Thanks & updated!
Btw is there any `cheatsheet` or nice overview of all these ddox magics`?